src/variables.js

```diff
@@ -4,8 +4,8 @@ export const ENVIRONMENT = process.env.NODE_ENV;
 let benefitservice_env_url = process.env.REACT_APP_BENEFITSERVICE_BASE_URL;
 
 if (!benefitservice_env_url && ENVIRONMENT !== "production") {
-  //  benefitservice_env_url = "http://localhost:1337";
-  // this uses the published version of strapi until we get the benefit service working.
+  benefitservice_env_url = "http://localhost:1337";
+  benefitservice_env_url = "https://benefit-service-dev.dev.dts-stn.com";
```

Comment:
This is not related to keycloak. Why is this being changed in this PR

Reply:
this is what it is on main, mostly

Reply:
regardless your branch is touching code that are not part of what you are implementing. If its too hard to revert recommend taking your components and staring again from main.